hetio/neo4j.py

```diff
@@ -165,7 +179,6 @@ def cypher_path(metarels):
 def construct_dwpc_query(metarels, property='name', join_hint='midpoint', index_hint=False, unique_nodes=True):
     """
     Create a cypher query for computing the *DWPC* for a type of path.
-
```

Comment:
Do you want to just save the whitespace changes for the other PR that modifies the PDP / DWPC functions?

Reply:
Sure thing